### PR TITLE
Add node's util module

### PR DESCRIPTION
**Pull request by valueduser**
Fix for #3501

##### Checklist
- [ ] `npm run test` passes - 5 tests failing prior to changes; same 5 failing after changes.
- [x] changelog is added, `npm run add-change`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -146,4 +147,4 @@
     "helper-test": "mocha scripts/lib/test-helper.spec.js",
     "csm-functional-test": "mocha test/publisher/functional_test"
   }
-}
```
Comment (AllanZhengYP) [inline package.json:149]: very nit: unnecessarily removed new line.

**Comment by AllanZhengYP** [comment]
@valueduser The change looks good to me. Can you address the only nit issue before we ship it?

File: package.json (hunk 1 above)
Comment (valueduser) [reply package.json]: No problem!

**Comment by valueduser** [comment]
@AllanZhengYP Done!

**Review by AllanZhengYP — APPROVED** [review]

**Comment by tamascsaba** [comment]
@trivikr I really want to thank you. 👍

**Pull request merged**